pkg/tf2pulumi/convert/eject_test.go

```diff
@@ -74,6 +78,50 @@ func isTruthy(s string) bool {
 	return s == "1" || strings.EqualFold(s, "true")
 }
 
+func formatTestFile(src io.Reader, dst io.Writer, isExperimental bool) error {
+	scanner := bufio.NewScanner(src)
+	inIf := false
+	inElse := false
+	for scanner.Scan() {
+		line := scanner.Text()
+		if line == "#if EXPERIMENTAL" {
+			if inIf || inElse {
+				return fmt.Errorf("Saw #if while already in an if or else block")
+			}
+			inIf = true
+		}
+		if line == "#else" {
+			if !inIf {
+				return fmt.Errorf("Saw #else while not in an if block")
+			}
+			if inElse {
+				return fmt.Errorf("Saw #else while already in an else block")
+			}
+			inIf = false
+			inElse = true
+		}
+		if line == "#endif" {
+			if !(inElse || inIf) {
+				return fmt.Errorf("Saw #endif while not in an if or else block")
+			}
+			inIf = false
+			inElse = false
+		}
+
+		doWrite := (inIf && isExperimental) ||
+			(inElse && !isExperimental) ||
+			(!inIf && !inElse)
+
+		if doWrite {
+			_, err := dst.Write([]byte(line + "\n"))
+			if err != nil {
+				return err
+			}
+		}
+	}
```

Comment:
Let's forbid as much as we can now, since we don't have any backwards compatibility concerns. 
```suggestion
	}
	if inIf {
		return fmt.Errorf("Unclosed if statement")
	}	
	if inElse {
		return fmt.Errorf("Unclosed else statement")
	}
```